package/setup.cfg

```diff
@@ -6,6 +6,8 @@
 ## Uncomment the following two lines to get the _release_ behavior.
 #use_cython=False
 #keep_cythonized=True
+## Uncomment the below line to generate the annotated cython HTML output
+# annotate_cython=True
```

Comment:
Didn't we merge this in already?

Reply:
Yes idk why it's showing up in diff.

Reply:
Might want to try rebasing, I guess we'll squash merge anyways, but it might get annoying for you if you get too out of synch :/